README.md

```diff
@@ -131,6 +131,12 @@ block.summary #=>
 # }
 ```
 
+#### Notes
```

Comment:
I think this deserves to have a bigger heading. If you change this from `#### Notes` to `## Notes` it would be better. Right now semantically it looks like `Notes` is nested under `Block`, but in fact this should apply to all.